public/js/question.js

```diff
@@ -44,4 +44,7 @@ var sendAnswerRequest = function(ans) {
             swal('Incorrect', 'Sorry, that\'s the wrong answer', 'error');
         }
     });
+
+    const numberOfAttempts = $('#attempts');
+    numberOfAttempts.html(parseInt(numberOfAttempts.html()) + 1);
```

Comment:
$('#attempts').html(parseInt($('#attempts').html()) + 1);

Reply:
I wonder if Async would affect this. You can move this inside the error function to make sure this updates after the submit fails.

Reply:
the variable is being called twice in this case making it inefficient, one call to get the object is more efficient

Reply:
it should also submit if it passes

Reply:
I am really confused about you definition of efficient. Anyway, use the complete function instead of updating it outside

Reply:
what do you mean by complete function?

Reply:
just like error:function.... you can do complete:function....
its the same as a final statement in a try catch block

Reply:
gotcha